packages/dev/core/src/FlowGraph/flowGraphSignalConnection.ts

```diff
@@ -1,23 +1,23 @@
 import type { Nullable } from "../types";
 import type { FlowGraphExecutionBlock } from "./flowGraphExecutionBlock";
-import { FlowGraphConnectionPointRole } from "./flowGraphConnectionPointRole";
+import { FlowGraphConnectionType } from "./flowGraphConnectionType";
 
 /**
  * @experimental
  * Represents a connection point for a signal.
  * When an output point is activated, it will activate the connected input point.
  * When an input point is activated, it will execute the block it belongs to.
  */
-export class FlowGraphSignalConnectionPoint {
-    private _connectedPoint: Nullable<FlowGraphSignalConnectionPoint>;
+export class FlowGraphSignalConnection {
+    private _connectedPoint: Nullable<FlowGraphSignalConnection>;
 
-    constructor(public name: string, public role: FlowGraphConnectionPointRole, private _ownerBlock: FlowGraphExecutionBlock) {}
+    constructor(public name: string, public role: FlowGraphConnectionType, private _ownerBlock: FlowGraphExecutionBlock) {}

```

Comment:
Might be good to do a search for `role` to make sure they are all changed.

```suggestion
    constructor(public name: string, public type: FlowGraphConnectionType, private _ownerBlock: FlowGraphExecutionBlock) {}
```